source/concepts/applications.rst

```diff
@@ -231,52 +176,106 @@ The command does not return anything. This leads to the following signature:
    multiuser/radially-orient-origins(radius = 1.0) -> None
 
 Let a set of players :math:`P = \{P_0, P_1, ... P_{N - 1}\}`, :math:`N` the number of
-players, and :math:`r` the radius given in argument. Then the center's position
-:math:`\mathbf{C}_i` for avatar :math:`i` is computed using polar coordinates converted
-to Cartesian. Each avatar is assigned an angle :math:`\theta_i`:
+players, and :math:`r` the radius given as an argument. Then the center's position
+:math:`\mathbf{C}_p` for avatar :math:`p` is computed using polar coordinates converted
+to Cartesian. Each avatar is assigned an angle :math:`\theta_p`:
 
 .. math::
 
-  \theta_i = \frac{i \times 2 \pi}{N}
+  \theta_p = \frac{ 2 \pi p}{N}
 
-Then the positions is:
+Then each position is:
 
 .. math::
 
   \begin{align}
-  \mathbf{C}_i &= \begin{bmatrix}
-    r\cos{\theta_i}\\
+  \mathbf{C}_p &= \begin{bmatrix}
+    r\cos{\theta_p}\\
     0\\
-    r\sin{\theta_i}\\
+    r\sin{\theta_p}\\
   \end{bmatrix}
   \end{align}
 
-The rotation :math:`\mathbf{R}_i` is expressed as a quaternion and is defined as:
+The rotation :math:`\mathbf{R}_p` is expressed as a quaternion and is defined as:
```

Comment:
Maybe continue this on from the previous sentence, to make it clear that it is the position and rotation of the same point in space.